tests/metrics/network/network-latency.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# Description:
+# This will measure latency when we do a ping
+# from one container to another (docker <-> docker)
+
+set -e
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+
+source "${SCRIPT_PATH}/../../lib/test-common.bash"
+
+# Image name (ping installed by default)
+image=busybox
+# Number of packets (sent)
+number=10
+
+function setup {
+	runtime_docker
+	kill_processes_before_start
+}
+
+# This script will perform all the measurements using a local setup
+
+# Test latency docker<->docker using ping
+
+function latency {
+	setup
+	latency_result=$(mktemp)
+	container_name="first_container"
+	second_container_name="second container"
+	$DOCKER_EXE run -tid --name ${container_name} ${image} sh > /dev/null
+	ip_address=$($DOCKER_EXE inspect --format "{{.NetworkSettings.IPAddress}}" $($DOCKER_EXE ps -ql))
```

Comment:
Rather than using the `docker ps -qa` here, can you not use the `${container_name} - which feels more correct and robust.
A local tests showed it to work for me.